modules/ROOT/pages/syntax/expressions.adoc

```diff
@@ -1302,36 +1290,55 @@ RETURN val, val IS NOT :: STRING AS notString
 [[type-predicate-expressions-null]]
 === Type predicate expressions for null
 
-All Cypher types include the `null` value.
-Therefore, `IS ::` will return `true` for all expressions evaluating to `null`, regardless of the type.
+A Cypher type will include the `null` value unless it is appended explicitly with `NOT NULL`.
+Therefore, `IS ::` will return `true` for all expressions evaluating to `null`, which do not explicitly remove it.
```

Comment:
```suggestion
Therefore, `IS ::` returns `true` for all expressions evaluating to `null`, unless they do not explicitly remove it.
```

The usage of the explicit `they` clears the ambiguity of whether `which` refers to `null`, as its default in grammar is, or to a previous term `expressions` (as intended in this case).

Reply:
This no longer feels natural in English. `Unless` is a negating condition and that alongside the `do not` is hard to read. I think it might even make the sentence incorrect.

Suggestion:

Therefore, `IS ::` returns `true` for all expressions evaluating to `null`, unless they explicitly remove it.

Reply:
You are right, the `not` slipped there from the initial sentence. Do take it away :)